docs/StyleGuide.md

```diff
@@ -1046,9 +977,9 @@ the rules below, it is good practice to update the code's style to match them.
 
 18. __Records__:
 
-    For records we often use `NamedFieldPuns` to make it convenient to
-    extract fields from the record. We also tend to use `RecordWildCards`
-    when the fields of the record are all of the form
+    For records we often use `NamedFieldPuns` to make it convenient to extract
+    fields from the record. We use the following convention when naming fields
+    to avoid duplicate record fields (we do not use `DuplicateRecordFields`):
```

Comment:
I second [@jorisdral's suggestion](https://github.com/input-output-hk/ouroboros-consensus/pull/24/files#r1164045079). Should we mention that we discourage the use of `RecordWildCards`?